deploy-worker/render_worker.sh

```diff
@@ -193,13 +193,17 @@ index=0
 
 for EDGE in ${ALLEDGECLUSTERS}; do
     create_worker_definitions ${EDGE} ${index}
-    WORKER_AGENT=$(oc --kubeconfig=${KUBECONFIG_HUB} get agent -n ${EDGE} --no-headers | grep worker | awk '{print $1}')
-    if [ "x${WORKER_AGENT}" == "x" ]; then
-        echo "Worker agent is empty for ${EDGE}, sleeping a bit more... 120 secs"
-        sleep 120
+    for a in {1..10}; do
```

Comment:
```suggestion
    echo "INFO: Waiting for Worker agent ( max 60 minutes) "
    for a in {1..10}; do
```

Reply:
this is a backport. Let's fix it in main and then backport it